app/javascript/src/components/Dashboard/Contacts/Pane/CreateContact.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui/v2";
+
+import formInitialValues from "constants/formInitialValues";
+
+import Form from "./Form";
+
+const NewContact = ({ fetchContacts, showPane, setShowPane, roles }) => {
+  const onClose = () => setShowPane(false);
+
+  return (
+    <Pane isOpen={showPane} onClose={onClose} size="lg">
+      <Pane.Header>
+        <Typography style="h2" weight="semibold">
+          Add New Contact
+        </Typography>
+      </Pane.Header>
+      <Form
+        onClose={onClose}
+        refetch={fetchContacts}
+        contact={formInitialValues.contactForm}
+        roles={roles}
+        isEdit={false}
+      />
+    </Pane>
+  );
+};
+
+export default NewContact;
```

Comment:
Please don't name or suffix file/folder/component with `Pane, Modal, Form or Page`.

Reply:
Please follow the same technique mentioned in `Contacts` PR.